schemas/oic.r.session-request.json

```diff
@@ -0,0 +1,36 @@
+{
+  "$schema" : "http://json-schema.org/draft-04/schema#",
+  "description" : "Copyright (c) 2017 Open Connectivity Foundation, Inc. All rights reserved.",
+  "title" : "Account Session",
+  "id": "https://www.openconnectivity.org/ocf-apis/security/schemas/oic.r.session-request.json#",
+  "definitions" : {
+    "uuid" : {
```

Comment:
Same as above. Reference the core schema.

Reply:
Done.